fixtures-service/json2sql.py

```diff
@@ -0,0 +1,40 @@
+import json_helpers
+
+import argparse
+import datetime
+import os
+
+
+def timestamp():
+    epoch = datetime.datetime.utcfromtimestamp(0)
+    now = datetime.datetime.now()
+    return round((now - epoch).total_seconds() * 1000)
+
+
+def main(input, migration_name, output_folder, write_mode):
+    json_data = json_helpers.load(input)
+
+    output_filename = os.path.join(output_folder, f'{timestamp()}_{migration_name}.up.sql')
```

Comment:
This is also to enforce consistency with hasura fixtures.

Reply:
They are independent. Hasura migrations are split into purely structural and purely data migrations. With hasura, btw, I was not able to generate a data migration. I always write them myself by hand.